pmlc/rt/vulkan/wrappers.cc

```diff
@@ -113,17 +119,17 @@ struct Registration {
     using pmlc::rt::registerSymbol;
 
     // Vulkan Runtime functions
-    registerSymbol("initVulkan", reinterpret_cast<void *>(initVulkan));
-    registerSymbol("deinitVulkan", reinterpret_cast<void *>(deinitVulkan));
-    registerSymbol("_mlir_ciface_createVulkanLaunchKernelAction",
-                   reinterpret_cast<void *>(createVulkanLaunchKernelAction));
-    registerSymbol("createVulkanMemoryTransferAction",
-                   reinterpret_cast<void *>(createVulkanMemoryTransferAction));
-    registerSymbol("setVulkanLaunchKernelAction",
-                   reinterpret_cast<void *>(setVulkanLaunchKernelAction));
-    registerSymbol("addVulkanLaunchActionToSchedule",
-                   reinterpret_cast<void *>(addVulkanLaunchActionToSchedule));
-    registerSymbol("run", reinterpret_cast<void *>(run));
+    registerSymbol("vkInit", reinterpret_cast<void *>(vkInit));
+    registerSymbol("vkDeinit", reinterpret_cast<void *>(vkDeinit));
+    registerSymbol("vkCreateLaunchKernelAction",
+                   reinterpret_cast<void *>(vkCreateLaunchKernelAction));
+    registerSymbol("vkCreateMemoryTransferAction",
+                   reinterpret_cast<void *>(vkCreateMemoryTransferAction));
+    registerSymbol("vkSetLaunchKernelAction",
+                   reinterpret_cast<void *>(vkSetLaunchKernelAction));
+    registerSymbol("vkRun", reinterpret_cast<void *>(vkRun));
+    registerSymbol("VkWait", reinterpret_cast<void *>(VkWait));
+    registerSymbol("VkScheduleFunc", reinterpret_cast<void *>(VkScheduleFunc));
```

Comment:
We should try to make the runtime functions consistent, I notice that some are `VkXXX` and others are `vkXXX`